### PR TITLE
Added RoboPlay swag

### PR DESCRIPTION
**Pull request by Pkmmte**
Reference: https://github.com/swapagarwal/swag-for-dev/issues/1067

- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Updated `data.json` with RoboPlay swag. Make something with **[Robo.js](https://github.com/Wave-Play/robo.js)** to get free stuff!

### CODE REVIEW

**Comment by 0xMouiz** [comment]
Woah, I am so excited!

**Comment by Nazeofel** [comment]
I'm excited for it ! SWAG WILL BE **MINEEEEEEEEEEEEEEEEEE**.

File: data.json [hunk 1]
```diff
@@ -593,6 +593,15 @@
     "dateAdded": "2020-09-03T15:43:00.000Z",
     "tags": ["stickers"]
   },
+  {
+    "name": "RoboPlay",
+    "difficulty": "medium",
+    "description": "Create a Discord bot or plugin with Robo.js - score 3 months of hosting, a month of Nitro, and a special edition T-Shirt!",
```
Comment (aslafy-z) [inline data.json:599]: Please adapt the wording to emphasize that swag will not be given to everyone but to the selected contributions only.

Comment (Pkmmte) [reply data.json]: @aslafy-z Just updated it. Does this new wording look better? :)

**Comment by aslafy-z** [comment]
Sorry, we choose to reject contests in this list. I forgot about it. Feel free to contribute for another swag!

**Comment by Pkmmte** [comment]
Understood, but may I ask why? 

It's our first time doing giveaways like these and the limit only exists due to limited funds, given we don't know how many people to expect.

**Comment by aslafy-z** [comment]
It is an historical rule that we set for this repository. We only pushed options that anyone can qualify for without 
 any kind of competition. We may revise this, wdyt @swapagarwal?

**Comment by aslafy-z** [comment]
Sure, I've will re-open the PR. Feel free to update the description and we can proceed with a merge :)

**Comment by Pkmmte** [comment]
Rebased and updated! 👍

**Pull request merged**